python/dolfinx/fem/function.py

```diff
@@ -485,7 +480,7 @@ def copy(self) -> Function:
     @property
     def x(self) -> la.Vector:
         """Vector holding the degrees-of-freedom."""
-        return la.Vector(self._cpp_object.x)  # type: ignore
+        return self._x  # type: ignore
```

Comment:
```suggestion
        return self._x
```
Note sure about this change - mypy will tell us if removing the 'ignore' is ok or nor.